### ISSUE TITLE
cassandra step documentation

### ISSUE THREAD

**Issue by timatim** [monoidspace/monoid#36]
### System Info

Mac OS M2, docker 24.0.7

### Description

Got `docker: Error response from daemon: network cass-network not found.` when running the cassandra command.

Would need to run `docker network create cass-network` first?

**Comment by adesokanayo** [comment]
Yes,  you will need to create the network first.  Better still we will switch to docker compose soon
